src/__test__/TodoList.test.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable import/extensions */
+/** * @jest-environment jsdom */
+import Methods from '../modules/Methods';
+import addtasklocalstorage from '../modules/localstorage';
+import Task from '../modules/constructor';
+import clearallsuccess from '../modules/clearallsucces';
+
+describe('add information in localstorage corretly ', () => {
+  const obj1 = new Task('test 1', 1);
+  const obj2 = new Task('test 2', 2);
+  const obj3 = new Task('test 3', 3);
+
+  test('emptytask', () => {
+    const tasks = addtasklocalstorage(obj1);
+    expect(tasks).toBe(1);
+  });
+
+  test('not empty task', () => {
+    addtasklocalstorage(obj2);
+    const tasks = addtasklocalstorage(obj3);
+    expect(tasks).toBe(3);
+  });
+});
+
+describe('Testing Add and delete element in form list', () => {
+  test('if list is not empty', () => {
+    const methods = new Methods();
+    const obj1 = new Task('test 1', 1, false);
+    const obj2 = new Task('test 2', 2, true);
+    document.body.innerHTML = '<div id="listtask">'
+            + ' <ul id="list"><li></li></ul>'
+            + '<div id="listtext"></div>';
+    methods.createelement(obj1);
+    methods.createelement(obj2);
+    const list = document.querySelectorAll('#list li');
+    expect(list).toHaveLength(1);
+  });
+});
+
+describe('Checkbox', () => {
+  test('Expect to change completed to tru after click', () => {
+    clearallsuccess();
+  });
+});
```

Comment:
- [ ] As they are [mandatory requirements ](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/tests_2.md)for this project, 

Create a test file(s) ([..].test.js) for a file(s) containing the following functions:
 - a function for editing the task description.
 - the "Clear all completed" function.

Please kindly add the above mentioned tests

Reply:
Hello @cvalencia1991 , @OsvaldoBC ,  please  I mistakenly approved this project, please make sure you implement the requested changes and resubmit for another review. I will be marked as require changes on your dashboards. ***Please do not merge this PR***